bin/common

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -e
+
```

Comment:
This does not seem to help. The best practice seems to suggest don't use `set -e`:

http://mywiki.wooledge.org/BashFAQ/105